VMAccess/vmaccess.py

```diff
@@ -275,6 +276,15 @@ def _set_user_account_pub_key(protect_settings, hutil):
 
     # Reset ssh key with the new public key passed in or reuse old public key.
     if cert_txt:
+        # support for SSH2-compatible format for public keys in addition to OpenSSH-compatible format
+        if (cert_txt.strip().startswith(BeginSSHTag)):
+            ext_utils.set_file_contents("temp.pub", cert_txt.strip())
+            retcode, output = ext_utils.run_command_get_output(['ssh-keygen', '-i', '-f', 'temp.pub'])
+            if retcode > 0:
+                raise Exception("Failed to convert SSH2 key to OpenSSH key.")
+            cert_txt = output
+            os.remove("temp.pub")
+
```

Comment:
nit: might be helpful to add logging message here that we've successfully converted the SSH2 key to OpenSSH